config/locales/en.yml

```diff
@@ -593,6 +593,10 @@ en:
       received_date: Date received at MOJ
       received_date_copy: "For example, 01 02 2018"
       case/ico:
+        external_deadline_copy: >-
+          We'll use this date to set the draft deadline, calculated as 20
+          working days from the final deadline.<br>
+          You will be able to change both dates later.
```

Comment:
Not sure I agree with this copy.  This sounds like the draft deadline is 20 workings after the final deadline. Perhaps we should remove this because I think we were going to allow users to input their own draft deadline.

Reply:
I'm sure Mia would be open to better wording of this. I think we went over it once together and I said the same thing, we can have a look tomorrow.